reference-recipes.js

```diff
@@ -58,5 +58,13 @@ module.exports = [
     {
         "title": `Right-to-left`,
         "path": "/resources/recipes/rtl/"
+    },
+    {
+        "title": `Flyout Menus`,
+        "path": "resources/recipes/flyout-menu"
+    },
+    {
+        "title": `Persistent Storage Migration`,
+        "path": "resources/recipes/persistent-storage-migration"
     }
 ]
```

Comment:
@harshitbhatt108  Please leave one empty line at the bottom

Reply:
Could you please change the image of the flyout, using a plugin that has a page full of UI / contents?